ci/builders/mac_host_engine.json

```diff
@@ -275,13 +272,49 @@
                     "--strip"
                 ],
                 "script": "flutter/sky/tools/create_macos_framework.py"
+            },
+            {
```

Comment:
Umm shall we add Release-FlutterMacOS.framework as well?

Reply:
umm asking because I assume Release-FlutterMacOS.framework would be the target to produce darwin-x64-release/FlutterMacOS.framework.zip?

Reply:
It is already there.